upstream-community-operators/mongodb-enterprise/1.5.5/mongodb-enterprise.v1.5.5.clusterserviceversion.yaml

```diff
@@ -0,0 +1,396 @@
+                    fieldRef:
+                      fieldPath: metadata.annotations['olm.targetNamespaces']
+                - name: MANAGED_SECURITY_CONTEXT
+                  value: 'true'
+                - name: OPERATOR_ENV
+                  value: prod
+                - name: MONGODB_ENTERPRISE_DATABASE_IMAGE
+                  value: quay.io/mongodb/mongodb-enterprise-database:1.5.5
+                - name: IMAGE_PULL_POLICY
+                  value: Always
+                - name: OPS_MANAGER_IMAGE_REPOSITORY
+                  value: quay.io/mongodb/mongodb-enterprise-ops-manager
+                - name: OPS_MANAGER_IMAGE_PULL_POLICY
+                  value: Always
+                - name: APP_DB_IMAGE_REPOSITORY
+                  value: quay.io/mongodb/mongodb-enterprise-appdb
+                - name: INIT_OPS_MANAGER_IMAGE_REPOSITORY
+                  value: quay.io/mongodb/mongodb-enterprise-init-ops-manager
+                - name: INIT_OPS_MANAGER_VERSION
+                  value: 1.0.1-b202006191242
+                - name: INIT_APPDB_IMAGE_REPOSITORY
+                  value: quay.io/mongodb/mongodb-enterprise-init-appdb
+                - name: INIT_APPDB_VERSION
+                  value: 1.0.1-b202006191242
+                image: quay.io/mongodb/mongodb-enterprise-operator:1.5.5
+                imagePullPolicy: Always
+                name: mongodb-enterprise-operator
+                resources:
+                  limits:
+                    cpu: 200m
+                    memory: 100Mi
+                  requests:
+                    cpu: 100m
+                    memory: 50Mi
+              serviceAccountName: mongodb-enterprise-operator
+      permissions:
+      - rules:
+        - apiGroups:
+            - ""
+          resources:
+            - configmaps
+            - secrets
+            - services
+          verbs:
+            - get
+            - list
+            - create
+            - update
+            - delete
+            - watch
+        - apiGroups:
+            - apps
+          resources:
+            - statefulsets
+          verbs:
+            - create
+            - get
+            - list
+            - watch
+            - delete
+            - update
+        - apiGroups:
+            - mongodb.com
+          resources:
+            - mongodb
+            - mongodb/finalizers
+            - mongodb/status
+            - mongodbusers
+            - mongodbusers/status
+            - opsmanagers
+            - opsmanagers/status
+            - opsmanagers/finalizers
+          verbs:
+            - get
+            - list
+            - watch
+            - delete
+            - update
+        serviceAccountName: mongodb-enterprise-operator
+      - rules:
+        - apiGroups:
+          - ""
+          resources:
+          - configmaps
+          verbs:
+          - get
+        serviceAccountName: mongodb-enterprise-appdb
+      - serviceAccountName: mongodb-enterprise-database-pods
```

Comment:
@J0zi the tests have failed and I'm not too sure about the reason. The only thing I did differently this time was adding these "empty" serviceAccounts. Do you think that this is responsible of the tests not passing?